tests/test_zk_evm_erc20.py

```diff
@@ -101,48 +104,69 @@ async def test_erc20(
             evm_address=evm_contract_address, starknet_address=starknet_contract_address
         ).execute(caller_address=1)
 
+        print("ERC20 Contract initiation tx infos")
```

Comment:
initialization?

Reply:
So right now, the "constructor" is working in 2 stages. Deployment and running the initial constructor code. The ideal is to have the initial constructor code run during deployment time